br/pkg/utils/backoff.go

```diff
@@ -220,14 +220,6 @@ type pdReqBackoffer struct {
 	maxDelayTime time.Duration
 }
 
-func NewPDReqBackoffer() Backoffer {
```

Comment:
I'm afraid we cannot simply take it away. some tests rely on the behavior of this backoffer and it may looks like "stuck" when retry too many times on some scenarios.